packages/tfjs-node-helpers/src/testing/metrics/binary-crossentropy.ts

```diff
@@ -0,0 +1,21 @@
+import { metrics } from '@tensorflow/tfjs-node';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+
+export class BinaryCrossentropyMetricCalculator extends MetricCalculator {
+  public calculate({ trueValues, probabilities }: MetricCalculatorData): Metric {
+    const trueY = trueValues.as1D();
+    const predY = probabilities.as1D();
+    const valueTensor = metrics.binaryCrossentropy(trueY, predY);
+    const value = valueTensor.dataSync()[0];
```

Comment:
@VladimirGlinskii 

```suggestion
    const [value] = valueTensor.dataSync();
```

Also, it would be great if we could use the async variant of this call.